core/domain/email_manager.py

```diff
@@ -171,7 +171,7 @@ def _send_email(
         return
 
     raw_plaintext_body = cleaned_html_body.replace('<br/>', '\n').replace(
-        '<br>', '\n').replace('</p><p>', '</p>\n<p>')
+        '<br>', '\n').replace('<li>', '- ').replace('</p><p>', '</p>\n<p>')
```

Comment:
I think it would be better to replace `<li>` with `<li>-  ` rather than replacing it directly with hyphen. Because if we do direct replacement then intermediate text body will have `</li>` tags but no `<li>` tags. Does that sound good ?